oscore-cf/src/main/java/org/eclipse/californium/oscore/Error.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2018 RISE SICS and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *    Tobias Andersson (RISE SICS)
+ *    
+ ******************************************************************************/
+package org.eclipse.californium.oscore;
+
+/**
+ * 
+ * Central location for all potential error messages
+ *
+ */
+public final class Error {
```

Comment:
The term/name `Error` is already used by the Java API (https://docs.oracle.com/javase/7/docs/api/java/lang/Error.html ) . Therefore I would prefer to rename to `ErrorDescriptions`.

Reply:
I will make up another name, something that is also short.

Reply:
Of course.